src/components/NotificationBox/initialize-icon-definitions.ts

```diff
@@ -0,0 +1,10 @@
+const initializeIconDefinitions = async () => {
```

Comment:
I haven't noticed this pattern in our other re usable component.

What's it for?

Please share rational and if we're doing this here should we be doing it for all our re usable components?